extensions/amp-ad-network-adzerk-impl/0.1/amp-ad-network-adzerk-impl.js

```diff
@@ -151,6 +151,18 @@ export class AmpAdNetworkAdzerkImpl extends AmpA4A {
 
   /** @override */
   getAmpAdMetadata(unusedCreative) {
+    if (this.ampCreativeJson_.analytics) {
```

Comment:
should you just load the analytics extension?

Reply:
I prefer not to load the extension here. AmpA4A will load extension based on metadata, I think it's better to load all extensions together.

Reply:
Talked offline. It's not that straightforward to load extension here. Will keep the current approach.